src/redturtle/prenotazioni/restapi/services/booking/add.py

```diff
@@ -36,22 +42,17 @@ def reply(self):
             if force:
                 # TODO: in futuro potrebbe forzare anche la data di fine oltre al gate
                 # create ha un parametro "duration" che può essere usato a questo scopo
-
-                if not api.user.has_permission(
-                    "redturtle.prenotazioni: Manage Prenotazioni",
-                    obj=self.context,
-                ):
-                    msg = self.context.translate(
-                        _("You are not allowed to force the gate.")
-                    )
-                    raise BadRequest(msg)
                 gate = data.get("gate", None)
                 duration = int(data.get("duration", -1))
-                obj = booker.book(data=book_data, force_gate=gate, duration=duration)
+                obj = booker.book(
+                    data=book_data,
+                    force_gate=gate,
+                    duration=duration,
+                )
             else:
                 obj = booker.book(data=book_data)
         except BookerException as e:
-            raise BadRequest(str(e))
+            raise BadRequest(e) from e
```

Comment:
?

Reply:
i had this suggestion in vscode: https://pylint.readthedocs.io/en/latest/user_guide/messages/warning/raise-missing-from.html